configs/mmdet/instance-seg/instance-seg_onnxruntime_dynamic_solo.py

```diff
@@ -0,0 +1,4 @@
+_base_ = [
```

Comment:
better change to algo name after task, for instance `instance-seg_solo_onnxruntime_dynamic.py`